docs/src/main/sphinx/release/release-391.md

```diff
@@ -0,0 +1,42 @@
+# Release 391 (20 July 2022)
+
+## General
+
+* Improve performance of repeated aggregations with ``CASE`` expressions. ({issue}`12548`)
+* Improve query performance when calling ``setBytes`` within a
```

Comment:
"synchronization section" is not something users understand. What are the user-visible conditions under which performance will improve? @sopel39